apps/organisations/models.py

```diff
@@ -28,13 +28,15 @@ class Organisation(models.Model):
         max_length=100,
         default='Organisation',
         help_text=_('The title of your organisation will be shown '
-                    'on the landing page. max. 100 characters')
+                    'on the landing page. max. 100 characters'),
+        blank=True
```

Comment:
I think the organisation name should be required

Reply:
But this is the title, the name is still required..I agreed on that with Katharina, see https://taiga.liqd.net/project/liqd-liquid-software/us/2796?milestone=214

Reply:
ok cool, maybe we can add required to the html or something